Libraries/LibFileSystem/FileSystem.cpp

```diff
@@ -10,6 +10,8 @@
 #include <LibCore/DirIterator.h>
 #include <LibCore/System.h>
 #include <LibFileSystem/FileSystem.h>
+// Required for Windows build
+#include <dirent.h>
```

Comment:
Wouldn't it be even more clear that it's Windows-specific if we include it if `AK_OS_WINDOWS` is defined? See two lines below this

Reply:
I wanted to do that first, but when I supposed that this might break Swift bindings

Reply:
I meant including it as an `#elif defined` in the existing block, similar to the other platforms.

Reply:
Like this?